build.gradle

```diff
@@ -53,3 +56,174 @@ subprojects {
         }
     }
 }
+
+/**
+ * This task will copy all of the source code in the `library` module into the `library-v3` module
+ * and change its imports from the Maps SDK on Play Services to the Standalone V3 Beta Maps SDK.
+ */
+task copyGmsToV3Beta(type: Copy) {
+    group 'Custom'
+    description 'Generates source code for Maps SDK for Android V3 BETA Utilities'
+
+    from('library/src/main/java') {
+        filter { line ->
+            line.replace('com.google.android.gms.maps', 'com.google.android.libraries.maps')
+        }
+        into 'main/java'
```

Comment:
For each copy, should this be of the pattern `library-v3/src/main/java`?

Or does it pick up the module path elsewhere?